python/trifinger_simulation/gym_wrapper/envs/finger_push.py

```diff
@@ -8,53 +8,72 @@
 from trifinger_simulation.gym_wrapper.finger_spaces import FingerSpaces
 from trifinger_simulation.gym_wrapper import utils
 from trifinger_simulation import (
+    finger_types_data,
     collision_objects,
     visual_objects,
     sample,
 )
 
 
 class FingerPush(gym.Env):
-    """
-    A gym environment to enable training on either the single or
-    the tri-fingers robots for the task of pushing
+    """A gym environment to enable training on any of the valid robots,
+    real or simulated, for the task of pushing.
     """
 
     def __init__(
         self, control_rate_s, finger_type, enable_visualization,
     ):
-        """
-        Constructor sets up the physical world parameters,
-        and resets to begin training.
+        """Intializes the constituents of the pushing environment.
+
+        Constructor sets up the finger robot depending on the finger type,
+        sets up the observation and action spaces, smoothing for
+        reducing jitter on the robot, and provides for a way to synchronize
+        robots being trained independently.
+
 
         Args:
-            control_rate_s (float): the rate at which the env step runs
-            finger-type (str "single"/"tri"): to train on the "single"
-                or the "tri" finger
+            control_rate_s (float): the rate at which step method of the env
+                will run. This is used to compute the number of times the same
+                action has to be applied to the robot in order to say that at
+                the end of this action loop, the robot would have reached the
+                desired action.
```

Comment:
I find this a bit confusing.  I think the following would be a bit clearer from a technical point of few (just a suggestion, though).

```suggestion
            control_rate_s (float): the rate (in seconds) at which step method of the env
                will run. The actual robot controller may run at a higher rate,
                so this is used to compute the number of robot control updates
                per environment step.
```

Reply:
ok, i agree